cowin/common/services/verloop_whatsapp_api.py

```diff
@@ -0,0 +1,48 @@
+import requests
+
+class VerloopWhatsappApi:
+
+    def __init__(self):
+        self.URL = "https://feedmyblr.verloop.io/api/v1/Campaign/SendMessage"
+        self.otp_campaign_id = ""
+        self.slots_campaign_id = ""
+
+    def send_otp(self, phone_number, variables):
+        body = {
+            "CampaignID": self.OTPCampaignID,
+            "To": {
+                "PhoneNumber": phone_number
+            },
+            "Variables": {
+                "name": variables["name"],
+                "otp": variables["otp"]
+            }
+        }
+        headers = {"Content-Type": "application/json"}
+        return self._send_msg(headers=headers, body=body)
+
+
+    def send_slot_availability(self, phone_number, variables):
+        body = {
+           "CampaignID": self.SlotsCampaignID,
+           "To": {
+               "PhoneNumber": phone_number
+           },
+           "Variables": {
```

Comment:
Same, check the templates that I have added in conversation. They're approved now.